src/base.js

```diff
@@ -0,0 +1,12 @@
+import Rebase from "re-base";
+import firebase from "firebase";
+
+export const firebaseApp = firebase.initializeApp({
+  apiKey: "AIzaSyC8ZnoxnKQPD43YTsgAKyWQdWvPQ5rbujM",
```

Comment:
Should this be public?

Reply:
IIRC, I think it's fine since firebase also limits by domain

Reply:
LOL I had the same question, and got yelled at by a github bot for uploading an API key. But yeah, the firebase devs say it's fine: https://github.com/zooniverse/vox/issues/32